Libplanet/Tx/ITxInvoice.cs

```diff
@@ -30,7 +31,7 @@ public interface ITxInvoice : IEquatable<ITxInvoice>
 
         /// <summary>
         /// A <see cref="HashDigest{SHA256}"/> value of the genesis which this transaction is made
-        /// from.  This can be <see langword="null"/> iff the transaction is contained in
+        /// from.  This can be <see langword="null"/> if the transaction is contained in
```

Comment:
I guess that iff isn't a typo. 😅  ([see also](https://en.wikipedia.org/wiki/If_and_only_if))